test/Npgsql.Tests/SecurityTests.cs

```diff
@@ -39,20 +39,16 @@ public void Default_user_uses_md5_password()
         }
 
         [Test, Description("Makes sure a certificate whose root CA isn't known isn't accepted")]
-        public void Reject_self_signed_certificate(
-            [Values(SslMode.VerifyCA, SslMode.VerifyFull)] SslMode sslMode,
-            [Values] bool checkCertificateRevocation)
+        public void Reject_self_signed_certificate([Values(SslMode.VerifyCA, SslMode.VerifyFull)] SslMode sslMode)
         {
             var connString = new NpgsqlConnectionStringBuilder(ConnectionString)
             {
                 SslMode = sslMode,
-                CheckCertificateRevocation = checkCertificateRevocation
+                CheckCertificateRevocation = false,
+                Pooling = false
```

Comment:
Maybe better to just use CreateTempPool (this way the test can be compatible with multiplexing, if we ever want to run that).